src/tooltip.js

```diff
@@ -86,6 +86,10 @@ angular.module('cp-tooltip')
 					tooltipDisplayed = false;
 					if(tooltipEl) tooltipEl.remove();
 				}
+
+				function encodeHTML(str) {
+					return str.replace(/&/g, '&amp;').replace(/</g, '&lt;').replace(/"/g, '&quot;');
+				}
```

Comment:
I'm just going to go with the original plan and use `$sanitize`, now that I think of it, I don't think it would be adding too much since any SPA that uses cp-tooltip probably already has ngSanitize